packages/common/test/app.e2e-spec.ts

```diff
@@ -21,4 +23,41 @@ describe('AppController (e2e)', () => {
       .expect(200)
       .expect('Hello World!');
   });
+
+  it('/files/upload-file (POST); for file with content', async () => {
+    const testFilePath = path.join(__dirname, 'empty.txt');
+
+    fs.writeFileSync(testFilePath, 'abcd');
```

Comment:
is this line required in the test case? doesn't seem to be getting used. If it is required, make sure to unlink the file as a test cleanup